### PR TITLE
Make a mistake of swapping db to the Realm

### PR DESCRIPTION
**Pull request by matbos**

### CODE REVIEW

File: Aplikacja/.idea/dictionaries/matt.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<component name="ProjectDictionaryState">
```
Comment (mgorski-zlatan) [inline Aplikacja/.idea/dictionaries/matt.xml]: remove this file. It's in gitignore ;)

File: Aplikacja/.idea/gradle.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mgorski-zlatan) [inline Aplikacja/.idea/gradle.xml]: remove file

File: Aplikacja/app/app.iml [hunk 3]
```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module external.linked.project.id=":app" external.linked.project.path="$MODULE_DIR$" external.root.project.path="$MODULE_DIR$/.." external.system.id="GRADLE" external.system.module.group="Aplikacja" external.system.module.version="unspecified" type="JAVA_MODULE" version="4">
```
Comment (mgorski-zlatan) [inline Aplikacja/app/app.iml]: remove file

File: Aplikacja/app/src/main/java/pl/lizardproject/qe2016/edititem/EditItemActivity.java [hunk 4]
```diff
@@ -18,15 +18,16 @@
...
 
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         Dart.inject(this);
 
         viewModel = new EditItemViewModel(itemId, this, ((MyApplication) getApplication()).getDatabaseFacade());
-        ((ActivityEditItemBinding) DataBindingUtil.setContentView(this, R.layout.activity_edit_item)).setViewModel(viewModel);
+        ((ActivityEditItemBinding) DataBindingUtil.setContentView(this, R.layout.activity_edit_item)).setViewModel(
+            viewModel);
```
Comment (mgorski-zlatan) [inline Aplikacja/app/src/main/java/pl/lizardproject/qe2016/edititem/EditItemActivity.java]: why new line?

Comment (matbos) [inline Aplikacja/app/src/main/java/pl/lizardproject/qe2016/edititem/EditItemActivity.java]: simply a formatter from other projects

File: Aplikacja/app/src/main/kotlin/pl/lizardproject/qe2016/itemlist/ItemListViewModel.kt [hunk 5]
```diff
@@ -1,28 +1,35 @@
...
-        items.addAll(values)
-    }
+    private var subscription: Subscription? = null
 
     val items = ObservableArrayList<Item>()
 
-    fun addItemCommand(view: View) {
-        activity.startActivity(Intent(activity, EditItemActivity::class.java))
+    fun addItemCommand(ignored: View) {
+        ignored.context.startActivity(
```
Comment (mgorski-zlatan) [inline Aplikacja/app/src/main/kotlin/pl/lizardproject/qe2016/itemlist/ItemListViewModel.kt]: rename 'ignored'

File: Aplikacja/app/src/main/java/pl/lizardproject/qe2016/edititem/EditItemActivity.java (hunk 4 above)
Comment (mgorski-zlatan) [inline Aplikacja/app/src/main/java/pl/lizardproject/qe2016/edititem/EditItemActivity.java]: Could you avoid line wrapping? :)

File: Aplikacja/app/src/main/kotlin/pl/lizardproject/qe2016/itemlist/ItemListViewModel.kt [hunk 6]
```diff
@@ -1,28 +1,35 @@
...
+                        .gotoEditItemActivity()
+                        .build())
     }
 
     fun dispose() {
-        subscription.unsubscribe()
+        subscription!!.unsubscribe()
+    }
+
+    fun loadData() {
```
Comment (mgorski-zlatan) [inline Aplikacja/app/src/main/kotlin/pl/lizardproject/qe2016/itemlist/ItemListViewModel.kt]: you can do it in constructor -> just as it was

**Comment by matbos** [comment]
@mgorski-zlatan please merge it unless you want to add some more comments.

**Pull request merged**